mkdocs.yml

```diff
@@ -58,6 +59,9 @@ markdown_extensions:
 
 plugins:
 - search
+- gen-files:
+    scripts:
+    - scripts/gen_doc_stubs.py
```

Comment:
What do you think about putting the script in the docs directory instead, just like I did for macros.py?

Reply:
Sure, I always do it that way. I started this commit before you added macros :sweat_smile: 

Note BTW that those 'macros.py' will end up browsable in the final site, but it's OK

Reply:
Ah, interesting. But like you said, it's OK.

Reply:
Done (this, and also moved the script)